lib/middleware.js

```diff
@@ -348,7 +348,7 @@ class Kevin {
             }
 
             // Mangle the url to get asset name
-            const assetName = this.getAssetName(reqPath);
+            const assetName = this.getAssetName(reqPath, req, res);
```

Comment:
Could you update the docs in the README to reflect this change?

Reply:
Done!